disnake/ext/commands/interaction_bot_base.py

```diff
@@ -106,12 +124,15 @@ def _app_commands_diff(
     "edit": "To edit:",
     "delete": "To delete:",
     "no_changes": "No changes:",
+    "delete_ignored": "Ignored due to delete flags:",
 }
 
 
-def _format_diff(diff: Dict[str, List[ApplicationCommand]]) -> str:
+def _format_diff(diff: _Diff) -> str:
     lines: List[str] = []
     for key, label in _diff_map.items():
+        if key not in diff:
+            continue
```

Comment:
Imo explicitly stating that no commands were added / edited / deleted is better than not printing anything

Reply:
It will still do that.

Reply:
Oh so `delete_ignored` is the only key that may not be present in a dict, I see